hugegraph-server/hugegraph-dist/pom.xml

```diff
@@ -216,6 +216,45 @@
                                 </target>
                             </configuration>
                         </execution>
+                        <execution>
+                            <id>download-keystore</id>
+                            <phase>prepare-package</phase>
+                            <goals>
+                                <goal>run</goal>
+                            </goals>
+                            <configuration>
+                                <target>
+                                    <exec executable="${shell-executable}"
+                                          dir="${project.basedir}"
+                                          failonerror="false">
+                                        <arg line="./download_keystore.sh"/>
```

Comment:
Perhaps you can modify the script name (`dist.sh`) executed in the above `download-swagger-ui` step, similar to what is done here, for differentiation.